Libs/DICOM/Widgets/Plugins/ctkDICOMWidgetsPlugins.h

```diff
@@ -31,7 +31,9 @@
 
 // CTK includes
 #include "ctkDICOMWidgetsPluginsExport.h"
+#ifdef HAVE_QT5
```

Comment:
The check should be done based on enabled/disabled features. We should avoid features disappearing silently based on detected Qt version.